package.json

```diff
@@ -47,21 +47,22 @@
     "dist"
   ],
   "dependencies": {
-    "@microsoft/signalr": "^8.0.0",
+    "@microsoft/signalr": "^8.0.7",
     "axios": "^1.4.0",
     "ts-log": "^2.2.5"
   },
   "devDependencies": {
-    "@types/node": "^20.14.7",
-    "@typescript-eslint/eslint-plugin": "^5.59.5",
-    "@typescript-eslint/parser": "^5.59.5",
-    "eslint": "^8.40.0",
-    "eslint-plugin-simple-import-sort": "^10.0.0",
-    "eslint-plugin-tsdoc": "^0.2.17",
-    "prettier": "^3.3.2",
-    "ts-node": "^10.9.1",
-    "tsup": "^8.0.2",
-    "typescript": "^5.0.4",
-    "vitepress": "^1.1.3"
+    "@types/node": "^22.7.0",
+    "@typescript-eslint/eslint-plugin": "^8.7.0",
+    "@typescript-eslint/parser": "^8.7.0",
+    "eslint": "^9.11.1",
+    "eslint-plugin-simple-import-sort": "^12.1.1",
+    "eslint-plugin-tsdoc": "^0.3.0",
+    "prettier": "^3.3.3",
+    "ts-node": "^10.9.2",
+    "tslib": "^2.7.0",
+    "tsup": "^8.3.0",
+    "typescript": "~5.5.4",
```

Comment:
Is there a reason we don't want to allow minor version bumps automatically anymore? They shouldn't be breaking by definition, right?

Reply:
According to eslint docs ts 5.6.0 is not yet supported even after two weeks, just to avoid weird issues

Reply:
ah - good call!